lambda/value_rec_compiler.ml

```diff
@@ -197,11 +197,12 @@ let compute_static_size lam =
     | Pbytes_set_64 _
     | Pbigstring_set_16 _
     | Pbigstring_set_32 _
-    | Pbigstring_set_64 _ ->
+    | Pbigstring_set_64 _
+    | Ppoll
         (* Unit-returning primitives. Most of these are only generated from
            external declarations and not special-cased by [Value_rec_check],
            but it doesn't hurt to be consistent. *)
-        Constant
+      -> Constant
```

Comment:
Style nitpick: this style is not consistent with the rest of this file. In addition, it is less clear that the comment applies to all the primitives and not only the last one.